webapp/src/js/directives/actionbar.js

```diff
@@ -42,6 +67,126 @@ angular.module('inboxDirectives').directive('mmActionbar', function() {
         $state.go('reports.detail', { id: null });
       };
 
+      ctrl.selectAll = () => {
```

Comment:
Maybe this should be a "reportsAction", along with the other actionbar actions that are connected to reports.

Reply:
I've moved this, and everything else in this controller, into the actions. I _think_ this makes sense but it feels strange. Probably just a matter of getting used to it.